collector-bitmovin-player/src/main/java/com/bitmovin/analytics/bitmovin/player/BitmovinPlayerCollector.java

```diff
@@ -32,8 +30,8 @@ public BitmovinPlayerCollector(
     }
 
     public void attachPlayer(BitmovinPlayer player) {
-        EventDataFactory factory = new EventDataFactory(this.bitmovinAnalyticsConfig, context, new DeviceInformationProvider(context, getUserAgent(context)), new UserIdProvider(context));
-        BitmovinSdkAdapter adapter = new BitmovinSdkAdapter(player, this.bitmovinAnalyticsConfig, factory,
+        DeviceInformationProvider deviceInformationProvider = new DeviceInformationProvider(context, getUserAgent(context));
```

Comment:
@Tigraine Short explanation: as we want the `EventData` creation to happen in `BitmovinAnalytics`, and not in the player adapter, we will remove the factory here for now (to keep things simple), and takle that in a separate PR.